recipe/meta.yaml

```diff
@@ -19,82 +19,83 @@
 {% set bump_daal_buildnum = "0" %}  # [not osx]
 {% set bump_daal_buildnum = "1" %}  # [osx]
 
+{% set tbb_version = "2021.7.0" %}
 
 # Linux
-{% set mkl_version = "2022.1.0" %}          # [linux]
-{% set mkl_fetch_buildnum = "223" %}        # [linux]
+{% set mkl_version = "2023.0.0" %}          # [linux]
+{% set mkl_fetch_buildnum = "25398" %}      # [linux]
 
 {% set openmp_version = mkl_version %}      # [linux]
-{% set openmp_fetch_buildnum = "3768" %}    # [linux]
+{% set openmp_fetch_buildnum = "25370" %}   # [linux]
 
-{% set dal_version = "2021.6.0" %}          # [linux]
-{% set dal_fetch_buildnum = "915" %}        # [linux]
+{% set dal_version = "2023.0.1" %}          # [linux]
+{% set dal_fetch_buildnum = "26646" %}      # [linux]
 
 {% set daal_version = "2021.5.3" %}         # [linux]
```

Comment:
this can be removed as there would be no new versions of daal package, only dal going forward

Reply:
well, not full true.  We need the daal one still as virual one, which points to the dal.  Otherwise some older stuff might get implicit incompatible

Reply:
Yes, but it won't be removed from repo if removed from recipe I assume. And there would be no package updates from Intel.

Reply:
Thank you @napetrov - we've removed the `daal*` outputs from this recipe.